### PR TITLE
Convert repository names to lowercase before referring to them as docker names.

### PR DESCRIPTION
**Pull request by LinZhihao-723**
# References

# Description
In PR (#187 ), we converted the repository names to lowercase before using them. However, there are still two actions using the unconverted repository names. This PR fixes the problem.

# Validation performed

### CODE REVIEW

File: .github/actions/clp-core-build-deps-and-binaries/action.yaml [hunk 1]
```diff
@@ -38,8 +38,16 @@ runs:
         local_registry_port: "${{inputs.local_registry_port}}"
         token: "${{inputs.token}}"
 
+    - id: "generate_image_id"
```
Comment (kirkrodrigues) [inline .github/actions/clp-core-build-deps-and-binaries/action.yaml:41]:
```suggestion
    - id: "sanitization"
```

File: .github/actions/clp-core-build-deps-and-binaries/action.yaml [hunk 2]
```diff
@@ -38,8 +38,16 @@ runs:
...
+      shell: "bash"
+      run: |
+        # Docker doesn't support repository names with uppercase characters, so we convert to
+        # lowercase here as the image id.
```
Comment (kirkrodrigues) [inline .github/actions/clp-core-build-deps-and-binaries/action.yaml:45]:
```suggestion
        # Docker doesn't support repository names with uppercase characters, so we convert the name
        # to lowercase here.
```

File: .github/workflows/clp-execution-image-build.yaml [hunk 3]
```diff
@@ -33,11 +33,19 @@ jobs:
           username: "${{github.actor}}"
           password: "${{secrets.GITHUB_TOKEN}}"
 
+      - id: "generate_image_id"
```
Comment (kirkrodrigues) [inline .github/workflows/clp-execution-image-build.yaml:36]:
```suggestion
      - id: "sanitization"
```

File: .github/workflows/clp-execution-image-build.yaml [hunk 4]
```diff
@@ -33,11 +33,19 @@ jobs:
...
+        shell: "bash"
+        run: |
+          # Docker doesn't support repository names with uppercase characters, so we convert to
+          # lowercase here as the image id.
```
Comment (kirkrodrigues) [inline .github/workflows/clp-execution-image-build.yaml:40]:
```suggestion
          # Docker doesn't support repository names with uppercase characters, so we convert the
          # name to lowercase here.
```

File: .github/actions/clp-core-build-deps-and-binaries/action.yaml [hunk 5]
```diff
@@ -38,8 +38,16 @@ runs:
...
+        echo "REPOSITORY=$(echo '${{github.repository}}' | tr '[:upper:]' '[:lower:]')" \
+          >> "$GITHUB_OUTPUT"
+
     - if: "inputs.deps_image_changed == 'false'"
       uses: "./.github/actions/clp-core-build"
       with:
-        image_name: >-
-          ghcr.io/${{github.repository}}/clp-core-dependencies-x86-${{inputs.os_name}}:main
+        image_name: "ghcr.io/${{steps.generate_image_id.REPOSITORY}}\
```
Comment (kirkrodrigues) [inline .github/actions/clp-core-build-deps-and-binaries/action.yaml:52]:
```suggestion
        image_name: "ghcr.io/${{steps.sanitization.REPOSITORY}}\
```

File: .github/workflows/clp-execution-image-build.yaml [hunk 6]
```diff
@@ -33,11 +33,19 @@ jobs:
...
+          echo "REPOSITORY=$(echo '${{github.repository}}' | tr '[:upper:]' '[:lower:]')" \
+            >> "$GITHUB_OUTPUT"
+
       - id: "meta"
         uses: "docker/metadata-action@v5"
         with:
-          images: >-
-            ${{env.CONTAINER_IMAGE_REGISTRY}}/${{github.repository}}/clp-execution-x86-ubuntu-focal
+          images: "${{env.CONTAINER_IMAGE_REGISTRY}}/${{steps.generate_image_id.REPOSITORY}}\
```
Comment (kirkrodrigues) [inline .github/workflows/clp-execution-image-build.yaml:47]:
```suggestion
          images: "${{env.CONTAINER_IMAGE_REGISTRY}}/${{steps.sanitization.REPOSITORY}}\
```

**Review by kirkrodrigues — CHANGES_REQUESTED** [review]

**Review by kirkrodrigues — APPROVED** [review]
Commit msg: `Convert repository names to lowercase before using them in docker image names. (#273)`?

**Comment by LinZhihao-723** [comment]
> Commit msg: `Convert repository names to lowercase before using them in docker image names. (#273)`?

LGTM

**Pull request merged**